core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

Comment:
@zvekovius -- my apologies for the delay, it's merged now. Please merge develop into your branch and let me know if it still doesn't work.

Also, note that you'll need to go to /admin, go to the config tab, and add your username to the list of users there in order to access the collection editor, otherwise you'll still get a 401.

Thanks!